### ISSUE TITLE
Chrome print function is broken for instant feature, print dialog opens and closes directly

### ISSUE THREAD

**Issue by ayeks** [squidfunk/mkdocs-material#1631]
__I checked that...__

- [x] ... the documentation does not mention anything about my problem
- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... there are no open or closed issues that are related to my problem

## Description

With versions 5.0.0, 5.1.0, 5.1.1 the Chrome Browsers print functionality is broken when using the `instant` feature. When you are on a docs page and press Ctrl+P the print dialog opens and closes directly. It works fine in Firefox with these versions.

The print dialog with mkdocs-material version 4.6.3 works fine. Probably because the instant feature has no effect there.

### Expected behavior

Expected behavior is that the print dialog of the browser opens and I can configure the print settings.

### Actual behavior

The print dialog opens and closes instantaneously. The print dialog works on any other side.

### Steps to reproduce the bug

1. Setup a completely new mkdocs project: `mkdocs new my-project`
2. Modify the mkdocs.yml file the following:

```yaml
site_name: My Docs
theme:
  name: material

theme:
  name: 'material'
  language: 'en'
  features:
    - instant
    - tabs
```

3. Run `mkdocs serve` and open the page in your browser.
4. Press `Ctrl+P` and the print dialog should open and close again directly.
5. Use the following mkdocs.yml file instead:

```yaml
site_name: My Docs
theme:
  name: material

theme:
  name: 'material'
  language: 'en'
  features:
    # - instant
    - tabs
```

6. Run `mkdocs serve` and open the page in your browser.
7. Press `Ctrl+P` and the print dialog should open.

### Package versions

* Python: `python --version`

```shell
Python 3.8.2
```

* MkDocs: `mkdocs --version`

```shell
mkdocs, version 1.1 from c:\git\docs\venv\lib\site-packages\mkdocs (Python 3.8)
```

* Material: `pip show mkdocs-material | grep -E ^Version`

```shell
Version: 5.1.1
```

### Project configuration

```yaml
site_name: My Docs
theme:
  name: material

theme:
  name: 'material'
  language: 'en'
  features:
    - instant
    - tabs
```

### System information

* OS: Windows 10
* Browser: Chrome Version 81.0.4044.113 (Offizieller Build) (64-Bit)

### Known Chrome Bug

I found a Chrome bug for that bug description: <https://bugs.chromium.org/p/chromium/issues/detail?id=467976>

> The window.print() followed by window.close() is the same problem reported on the bug linked above.

---

Do you have any idea why this bug exists for the instant feature?
Which additional tests can I perform to narrow down the problem?

**Comment by squidfunk** [comment]
Nope, never seen it so far. Unfortunately, I'm currently a little short on time, so if somebody else can investigate, that'd be awesome!

**Comment by squidfunk** [comment]
I can't reproduce it on Chrome 81, macOS:

![Ohne Titel](https://user-images.githubusercontent.com/932156/80310978-8fe6a100-87dd-11ea-835c-56aa50588c37.gif)

What's interesting is, that Chrome seems to reload the page before printing, which can be seen through the GitHub integration updating. Maybe someone encountering this issue can help narrow it down? I would suggest the following:

1. [Setup the development environment](https://squidfunk.github.io/mkdocs-material/customization/#environment-setup)
2. Enable instant loading and start [development mode](https://squidfunk.github.io/mkdocs-material/customization/#development-mode)
3. The whole instant loading code is located [here](https://github.com/squidfunk/mkdocs-material/blob/master/src/assets/javascripts/integrations/instant/index.ts), so maybe start by commenting out specific lines.

**Comment by ayeks** [comment]
Thanks for trying to reproduce the bug. I experience the issue on Chromium `Version 80.0.3987.163 (Offizieller Build) Built on Ubuntu , running on Ubuntu 18.04 (64-Bit)` as well. So at least the bug exists on Windows and Ubuntu.

I setup the development environment as you described it and experience the problem there as well. Are you sure that you activated the instant feature in the `mkdocs.yml` when you tested it? By default it is commented out.

Unfortunately I have no experience in Javascript, so I am of no use here. :/

**Comment by squidfunk** [comment]
I definitely turned on instant loading, as you can see from the non-reloading GitHub stars integration.

**Comment by squidfunk** [comment]
I'm closing this, as nobody else seems to experience the problem, so it might be related to your setup. If somebody manages to isolate the problem and provide a reproducible case, we can, of course, fix it.

**Comment by squidfunk** [comment]
Finally reproducible and fixed in #2261!
